src/app/rpmostree-override-builtins.cxx

```diff
@@ -68,8 +70,12 @@ handle_override (RPMOSTreeSysroot  *sysroot_proxy,
                  GError           **error)
 {
   glnx_unref_object RPMOSTreeOS *os_proxy = NULL;
-  if (!rpmostree_load_os_proxy (sysroot_proxy, opt_osname,
-                                cancellable, &os_proxy, error))
+  RPMOSTreeOSExperimental *osexperimental_proxy = NULL;
+  g_autoptr(GPtrArray) override_replace_final = g_ptr_array_new_with_free_func (free);
+  GUnixFDList *list_fds = g_unix_fd_list_new ();
```

Comment:
```suggestion
  g_autoptr(GPtrArray) override_replace_final = NULL;
  g_autoptr(GUnixFDList) list_fds = NULL;
```

And you can initialize `override_replace_final` down in the if block once we know we need it.